client/dist/styles.css

```diff
@@ -0,0 +1,8 @@
+#header{
+  background-color: rgb(82, 110, 103);
+  color: white;
+}
+
+h1{
```

Comment:
please be sure to add class names or ids, otherwise your css changes will impact the positioning of everyone's h1s.

something like h1.overview-header would help you only impact the elements that you intend.

Here is short cheat sheet: https://www.w3schools.com/cssref/css_selectors.asp